README.md

```diff
@@ -812,10 +812,11 @@ Zod                | ![Zod](https://img.shields.io/badge/zod-%233068b7.svg?style
 
 | Name      | Badge                                                                                                          | Markdown                                                                                                         |
 | --------- | -------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
-| Hibernate    | ![Hibernate](https://img.shields.io/badge/Hibernate-59666C?style=for-the-badge&logo=Hibernate&logoColor=white)          | `![Hibernate](https://img.shields.io/badge/Hibernate-59666C?style=for-the-badge&logo=Hibernate&logoColor=white)`          |
+| Doctrine  | ![Doctrine](https://img.shields.io/badge/doctrine-fa6a3c?style=for-the-badge&logo=doctrine&logoColor=white)    | `![Doctrine](https://img.shields.io/badge/doctrine-fa6a3c?style=for-the-badge&logo=doctrine&logoColor=white)`    |
+| Hibernate | ![Hibernate](https://img.shields.io/badge/Hibernate-59666C?style=for-the-badge&logo=Hibernate&logoColor=white) | `![Hibernate](https://img.shields.io/badge/Hibernate-59666C?style=for-the-badge&logo=Hibernate&logoColor=white)` |
```

Comment:
Kindly undo the change made to Hibernate.

Reply:
Done